ballerina/client_utils.bal

```diff
@@ -16,66 +16,37 @@
 
 import ballerina/http;
 import ballerina/url;
-
-# Provides settings related to HTTP/1.x protocol.
```

Comment:
Why are these removed?

Reply:
These are still relevant to the user since they set configurables. Please undo this change for client configuration.